gremlin-driver/docker-compose.yml

```diff
@@ -22,12 +22,5 @@ services:
   gremlin-socket-server:
     container_name: gremlin-socket-server
     image: tinkerpop/gremlin-socket-server:${GREMLIN_VERSION}
-    build:
```

Comment:
Actually, I think in this case, since we are just running the image, perhaps we can just remove the need for the compose file and execute the docker run command from pom file all together?